plugins/webdriver/pom.xml

```diff
@@ -4,197 +4,199 @@
 
     <groupId>kg.apc</groupId>
     <artifactId>jmeter-plugins-webdriver</artifactId>
-    <version>2.3</version>
+    <version>2.4</version>
 
     <name>WebDriver/Selenium</name>
-    <description>WebDriver/Selenium</description>
```

Comment:
Please remove all indentation changes, so only the actual XML changes will be visible.